presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -168,6 +168,10 @@ Property Name                                      Description
                                                    performance on tables with small files. A higher value may
                                                    improve performance for queries with highly skewed
                                                    aggregations or joins.
+
+``iceberg.enable-merge-on-read-mode``              Enable merge-on-read mode for Iceberg tables provided that        ``false``
```

Comment:
Suggest something like the following: 

"Enables merge-on-read mode for Iceberg tables when set to ``true``. 
Note: merge-on-read support is currently under development. Be aware of the risks involved.